client/src/components/beer-edit.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useCallback, useState } from "react";
```

Comment:
Please remove this file since it does nothing yet

Reply:
Thanks. I set it up to work on later but it's good to learn you should only commit actual features.